### ISSUE TITLE
Does `tag` have to be required when creating a message?

### ISSUE THREAD

**Issue by Powersource** [ssbc/ssb-buttwoo#5]
I was getting an error about `tag` missing as an opt, when I added that it seems to have fixed things. So now I have

```
ssb.db.create(
        {
          content,
          recps: recipientKeys,
          encryptionFormat: 'box2',
          feedFormat: 'buttwoo-v1',
          tag: 0,
        },
```

a bit verbose (and also confusing, what's a tag)?

**Comment by Powersource** [comment]
And yeah I get that the tag is what's described in https://github.com/ssbc/ssb-buttwoo-spec#metadata

**Comment by arj03** [comment]
Yeah tag is required. It would be good if this repo had at least an example of how to use it in ssb-db2 ;-)

It's not really super clear that it should be an int: https://github.com/ssbc/ssb-buttwoo/blob/master/format.js#L91

**Comment by arj03** [comment]
Also the enum is exported, so you can use `tag: butt2.tags.SSB_FEED` for improved readability.

**Comment by Powersource** [comment]
Could it default to SSB_FEED? And could it be called something like messageType instead?

**Comment by arj03** [comment]
I guess it would be good with a default. Can you make a PR for that? 

Maybe we can call it just type? It is just also quite generic. Hmm

**Comment by Powersource** [comment]
Tried to  use SSB_FEED but got

```
TypeError: Cannot read properties of undefined (reading 'SSB_FEED')
```

Does this comment have something to do with that?

https://github.com/ssbc/ssb-buttwoo/blob/b1f0a0585a07a3e64f0b3793cb200a646d57120b/format.js#L323-L326
